### PR TITLE
dao5 task implementation

### PR DESCRIPTION
**Pull request by less08**

### CODE REVIEW

File: src/main/java/core/dao/jdbc/ManufacturerDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        try (Connection connection = ConnectionUtil.getConnection();
+                  PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            Manufacturer manufacturer = null;
+            while (resultSet.next()) {
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                manufacturer = new Manufacturer(name, country);
+                manufacturer.setId(resultSet.getObject("id", Long.class));
```
Comment (vicire) [inline src/main/java/core/dao/jdbc/ManufacturerDaoJdbcImpl.java]: better to make private method for this logic

File: src/main/resources/init_db.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+CREATE SCHEMA `storage` DEFAULT CHARACTER SET utf8 ;
+CREATE TABLE `storage`.`manufacturers`
+(
+  id              INT unsigned NOT NULL AUTO_INCREMENT,
+  name            VARCHAR(150) NOT NULL,
+  country         VARCHAR(150) NOT NULL,
+  deleted         bit(1) NOT NULL DEFAULT b'0',
```
Comment (vicire) [inline src/main/resources/init_db.sql:7]: optional, but why do you choose bit and not tinyint?

File: src/main/java/core/util/ConnectionUtil.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't find MySQL Driver", e);
+        }
+    }
```
Comment (vicire) [inline src/main/java/core/util/ConnectionUtil.java:15]: Optional - you can delete this block. Or you can`t get a connection without him?

File: pom.xml [hunk 4]
```diff
@@ -18,6 +18,11 @@
     </properties>
```
Comment (vicire) [inline pom.xml]: remove empty line

**Review by vicire — CHANGES_REQUESTED** [review]

File: src/main/resources/init_db.sql (hunk 2 above)
Comment (less08) [reply src/main/resources/init_db.sql:7]: That's what i found on stackoverflow, and I thought BIT(1) suits better: 

> "A TINYINT is an 8-bit integer value, a BIT field can store between 1 bit, BIT(1), and 64 bits, BIT(64). For a boolean values, BIT(1) is pretty common."

File: src/main/java/core/util/ConnectionUtil.java (hunk 3 above)
Comment (less08) [reply src/main/java/core/util/ConnectionUtil.java:15]: I can, but this is how Bohdan did this in the video tutorial)

**Review by vicire — APPROVED** [review]

File: src/main/java/core/dao/jdbc/ManufacturerDaoJdbcImpl.java [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers ( name, country) VALUES(?,?)";
```
Comment (boroda4436) [inline src/main/java/core/dao/jdbc/ManufacturerDaoJdbcImpl.java]:
```suggestion
        String query = "INSERT INTO manufacturers (name, country) VALUES(?,?)";
```

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**